suite-native/module-send/src/components/CoinControl/UtxoCard.tsx

```diff
@@ -32,6 +33,10 @@ const AccountAddressFormatterStyle = prepareNativeStyle(() => ({
     maxWidth: '80%',
 }));
 
+const CardStyle = prepareNativeStyle(utils => ({
```

Comment:
nit: camelCase for consts please

```suggestion
const cardStyle = prepareNativeStyle(utils => ({
```

Reply:
nit resolved in ca5db0358ff795b5d1c4a82eb75b727594276e58